core/domain/interaction_registry.py

```diff
@@ -104,3 +118,13 @@ def get_all_specs(cls):
             interaction.id: interaction.to_dict()
             for interaction in cls.get_all_interactions()
         }
+
+    @classmethod
+    def get_all_question_editor_specs(cls):
+        """Returns a dict containing the full specs of each interaction."""
+        # pylint: disable=C0301
+        return {
+            interaction.id: interaction.to_dict()
+            for interaction in cls.get_all_interactions(is_editor_context_exploration=False)
```

Comment:
You don't need to pass a flag to get_all_interactions. Just use a new method.

Reply:
Okay.